product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -6,85 +6,85 @@ deepToC: true
 
-Upgrading your Postgres cluster to a newer major version is recommended to access improved features, performance enhancements, and security updates. This guide provides general step-by-step instructions and a worked example to help facilitate a smooth and controlled upgrade process.
+We recommend upgrading your Postgres cluster to a newer major version to access improved features, performance enhancements, and security updates. These step-by-step instructions and a worked example help to provide a smooth and controlled upgrade process.
 
 ### Plan the upgrade
 
-The cluster must be available throughout the process (i.e., a *rolling* upgrade). To achieve continuous cluster availability, you must [perform a planned switchover](#performing-a-planned-switchover) of the [write leader](https://www.enterprisedb.com/docs/pgd/latest/terminology/#write-leader) node before upgrading it so that there is always a write leader available. 
+The cluster must be available throughout the process (that is, a *rolling* upgrade). To achieve continuous cluster availability, you must [perform a planned switchover](#performing-a-planned-switchover) of the [write leader](https://www.enterprisedb.com/docs/pgd/latest/terminology/#write-leader) node before upgrading it so that a write leader is always available. 
 
-You can see which node is the write leader by connecting to the PGD cluster (e.g. using SSH) and running the `pgd show-groups` command:
+You can see which node is the write leader by connecting to the PGD cluster (for example, using SSH) and running the `pgd show-groups` command:
 
 ```bash
 sudo -u postgres pgd show-groups
 ```
 
-To keep the number of planned [switchovers](https://www.enterprisedb.com/docs/pgd/latest/cli/command_ref/pgd_switchover/) to a minimum, you should upgrade the current write leader last in your upgrade order, so you only have to perform [one planned switchover](#performing-a-planned-switchover) during the upgrade process.
+To keep the number of planned [switchovers](https://www.enterprisedb.com/docs/pgd/latest/cli/command_ref/pgd_switchover/) to a minimum, upgrade the current write leader last in your upgrade order, so you have to perform only [one planned switchover](#performing-a-planned-switchover) during the upgrade process.
 
 ### Perform the upgrade on each node
 
 !!! Note
-Ensure that your databases and configuration files are backed up before starting the upgrade process to help prevent data loss.
+To help prevent data loss, ensure that your databases and configuration files are backed up before starting the upgrade process.
 !!!
 
-Next, carry out the following steps on each node in order:
+Next, perform the following steps on each node in order:
```

Comment:
"carry out" is idiomatic and might not  be clear to non-native English speakers.